### PR TITLE
NullPointerException when attempting to download feedback results #4168

### LINKED ISSUE #4168: NullPointerException when attempting to download feedback results
Regression due to #3789

### PR DESCRIPTION
**Pull request by JosephineKwa**
Fixes #4168

### CODE REVIEW

**Comment by kanghj** [comment]
Add a test for this, perhaps in another issue

File: src/main/java/teammates/common/datatransfer/FeedbackContributionQuestionDetails.java [hunk 1]
```diff
@@ -381,7 +380,7 @@ public String getQuestionResultStatisticsCsv(
             String displayName = name;
             String displayTeam = team;
             String displayEmail = email;
-            if(hideRecipient == true && hiddenRecipients.contains(email)){
+            if(hideRecipient == true){
```
Comment (kanghj) [inline src/main/java/teammates/common/datatransfer/FeedbackContributionQuestionDetails.java]: if (hideRecipient) {

File: src/main/java/teammates/common/datatransfer/FeedbackContributionQuestionDetails.java [hunk 2]
```diff
@@ -329,7 +329,7 @@ public String getQuestionResultStatisticsCsv(
         responses = getActualResponses(question, bundle);
 
         //List of teams with at least one response
```
Comment (kanghj) [inline src/main/java/teammates/common/datatransfer/FeedbackContributionQuestionDetails.java]: wrong comment

**Comment by JosephineKwa** [comment]
@kanghj ready for review

**Comment by kanghj** [comment]
Ok looks fine to me

**Pull request merged**